docs/framework-integration/vue.md

```diff
@@ -281,24 +319,198 @@ createApp(App).use(ComponentLibrary).mount('#app');
+
+```tsx
+const componentModels: ComponentModelConfig[] = [
+  {
+    elements: ['my-input', 'my-textarea'],
+    event: 'v-on-change',
+    externalEvent: 'on-change',
+    targetAttr: 'value'
+  },
+];
+
+export const config: Config = {
+  namespace: 'stencil-library',
+  outputTargets: [
+    vueOutputTarget({
+      componentCorePackage: 'component-library',
+      proxiesFile: '{path to your proxy file}',
+      componentModels: componentModels,
+    })
+  ],
+};
+```
+
+### customElementsDir
+
+**Optional**
+
+**Default: 'dist/components'**
+
+**Type: `string`**
+
+If [includeImportCustomElements](#includeimportcustomelements) is `true`, this option can be used to specify the directory where the generated
+custom elements live. This value only needs to be set if the `dir` field on the `dist-custom-elements` output target was set to something other than
+the default directory.
+
+### excludeComponents
+
+**Optional**
+
+**Default: `[]`**
+
+**Type: `string[]`**
+
+This lets you specify component tag names for which you don't want to generate Vue wrapper components. This is useful if you need to write framework-specific versions of components. For instance, in Ionic Framework, this is used for routing components - like tabs - so that
+Ionic Framework can integrate better with Vue's Router.
+
+### includeDefineCustomElements
+
+**Optional**
+
+**Default: `true`**
+
+**Type: `boolean`**
+
+If `true`, all Web Components will automatically be registered with the Custom Elements Registry. This can only be used when lazy loading Web Components and will not work when `includeImportCustomElements` is `true`.
+
+### includeImportCustomElements
+
+**Optional**
+
+**Default: `undefined`**
+
+**Type: `boolean`**
+
+If `true`, the output target will import the custom element instance and register it with the Custom Elements Registry when the component is imported inside of a user's app. This can only be used with the [Custom Elements Bundle](/docs/custom-elements) and will not work with lazy loaded components.
+
+### includePolyfills
+
+**Optional**
+
+**Default: `true`**
+
+**Type: `boolean`**
+
+If `true`, polyfills will automatically be imported and the `applyPolyfills` function will be called in your proxies file. This can only be used when lazy loading Web Components and will not work when `includeImportCustomElements` is enabled.
+
+### loaderDir
+
+**Optional**
+
+**Default: `/dist/loader`**
+
+**Type: `string`**
+
+The path to where the `defineCustomElements` helper method exists within the built project. This option is only used when `includeDefineCustomElements` is enabled.
+
+### proxiesFile
+
+**Required**
+
+**Type: `string`**
+
+This parameter allows you to name the file that contains all the component wrapper definitions produced during the compilation process. This is the first file you should import in your Vue project.
+
 ## FAQ
 
+### Do I have to use the `dist` output target?
```

Comment:
Were you able to get `my-component` rendering without updating `vite.config.js` (as opposed to `vue.config.js`)?

Reply:
Hmmm I am having some issues when trying to use `includeImportCustomElements`. My knowledge of Vue is pretty limited, so I'll talk with Sean (I think he has some knowledge of the framework and how these wrappers work) or can figure out who might have the most insight here

Reply:
I just missed a step on my end. Was able to render correctly w/ `dist` and `dist-custom-elements` without changing the config

Reply:
Hmmm I do. I had to update my `vite.config.ts`. Let's take 30 minutes to walk through this together - I think I'm doing everything correctly, but who knows 🤣

Reply:
(I wanna block off that time as I would hope that going through the directions from top to right before the FAQ section should result in a working app. It's more than likely I'm missing something here, but if that's a result of me misinterpreting the instructions, I wanna figure out where that clarification point is).